dataStandard_versioned_docs/version-5/whats-new/api_changes_5.md

```diff
@@ -0,0 +1,349 @@
+* **PUT** /ed-fi/studentAssessmentRegistrationBatteryPartAssociations/{id}
+    Updates a specific student assessment registration battery part association by ID.
+* **DELETE** /ed-fi/studentAssessmentRegistrationBatteryPartAssociations/{id}
+    Deletes a specific student assessment registration battery part association by ID.
+* **GET** /ed-fi/studentAssessmentRegistrationBatteryPartAssociations/deletes
+    Retrieves deleted student assessment registration battery part associations based on change version.
+### studentEducationOrganizationAssessmentAccommodations
+* **GET** /ed-fi/studentEducationOrganizationAssessmentAccommodations
+    Retrieves specific student education organization assessment accommodations using property values.
+* **POST** /ed-fi/studentEducationOrganizationAssessmentAccommodations
+    Creates or updates student education organization assessment accommodations based on natural key values.
+* **GET** /ed-fi/studentEducationOrganizationAssessmentAccommodations/{id}
+    Retrieves a specific student education organization assessment accommodation by ID.
+* **PUT** /ed-fi/studentEducationOrganizationAssessmentAccommodations/{id}
+    Updates a specific student education organization assessment accommodation by ID.
+* **DELETE** /ed-fi/studentEducationOrganizationAssessmentAccommodations/{id}
+    Deletes a specific student education organization assessment accommodation by ID.
+* **GET** /ed-fi/studentEducationOrganizationAssessmentAccommodations/deletes
+    Retrieves deleted student education organization assessment accommodations based on change version.
+### studentSection504ProgramAssociations
+* **GET** /ed-fi/studentSection504ProgramAssociations
+    Retrieves specific student section 504 progam associations using property values.
+* **POST** /ed-fi/studentSection504ProgramAssociations
+    Creates or updates student section 504 progam associations based on natural key values.
+* **GET** /ed-fi/studentSection504ProgramAssociations/{id}
+    Retrieves a specific student section 504 progam association by ID.
+* **PUT** /ed-fi/studentSection504ProgramAssociations/{id}
+    Updates a specific student section 504 progam association by ID.
+* **DELETE** /ed-fi/studentSection504ProgramAssociations/{id}
+    Deletes a specific student section 504 progam association by ID.
+* **GET** /ed-fi/studentSection504ProgramAssociations/deletes
+    Retrieves deleted student section 504 progam associations based on change version.
+
+## Modified EndPoints
+### Add Dual Credit to studentSectionAssociation
+* **GET** /ed-fi/studentSectionAssociations 
+    Retrieves specific student section associations using property values.
+##### Parameters
+>   * **Add** dualCreditEducationOrganizationReference
+>   * **Add** dualCreditIndicator
+>   * **Add** dualCreditInstitutionDescriptor
+>   * **Add** dualCreditTypeDescriptor
+>   * **Add** dualHighSchoolCreditIndicator
```

Comment:
I would remove `>  ` everywhere in the file. The bullet points are fine here without that.